public/index.html

```diff
@@ -0,0 +1,493 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>AppBox</title>
+		<meta name="viewport" content="width=device-width,user-scalable=no,initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+		<link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+		<link href="https://fonts.googleapis.com/css?family=Roboto:400,300italic,500" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="css/gallery.build.css">
+    <link rel="stylesheet" href="css/gallery.theme.css">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+	<body>
+    <nav>
+      <img src="images/appbox-wh.svg" alt="AppBox Logo" class="appbox-logo">
+      <label for="show-menu" class="show-menu">MENU</label>
+      <input type="checkbox" id="show-menu" role="button">
+      <ul class="menu">
+        <li><a href="#home-nav">HOME</a></li>
+        <li><a href="#about-nav">ABOUT</a></li>
+        <li><a href="#features-nav">FEATURES</a></li>
+        <li><a href="#reviews-nav">REVIEWS</a></li>
+        <li><a href="#screens-nav">SCREENS</a></li>
+        <li><a href="#blog-nav">BLOG</a></li>
+        <li><a href="#support-nav">SUPPORT</a></li>
+        <li class="searchbox">
+         <input type="text" placeholder="Search......" name="search" class="searchbox-input" required>
+         <span class="icon-search"></span>
+       </li>
+      </ul>
+    </nav>
+		<section id="home-nav" class="best-way">
+      <div class="title-wrapper">
+        <div class="title-text">
+          <h1>Best Way To Present Your Beautiful Mobile App</h1>
+          <p>AppBox is full of awesome features. It's designed for <span>App Landing and App Store site</span></p>
+          <a href="#" class="highlight-link med-eclipse-border">LEARN MORE</a>
+        </div>
+        <img src="assets/head-iPhone.png" alt="iPhone">
+      </div>
+      <div class="get-it-free">
+        <h2 class="first-skew">GET IT FOR FREE</h2>
+        <p>Choose your native platform and download the app FREE!</p>
+        <div class="download-link-wrapper">
+          <a href="#" class="lg-eclipse-border">
+            <span class="icon-google-play">Google Play</span>
+          </a>
+          <a href="#" class="lg-eclipse-border">
+            <span class="icon-apple">iOS App Store</span>
```

Comment:
There really is no benefit to placing an icon on a span. In my experience it works just as fine when you apply to the parent element, the `a` in the example. 

It's not true in all cases, but I strive to NOT use an arbitrary DOM element for the sake of presentation if it's not absolutely needed.